applications/FluidDynamicsBiomedicalApplication/custom_utilities/parabolic_profile_utilities.cpp

```diff
@@ -199,15 +199,18 @@ void ParabolicProfileUtilities::ImposeParabolicInlet(
     const double MaxValueFactor)
 {
     // Impose the parabolic inlet profile
-    ImposeParabolicProfile(rModelPart, MaxParabolaValue, MaxValueFactor);
+    ImposeParabolicProfile(rModelPart, *MaxParabolaValue, MaxValueFactor);
 }
 
 template<class TInputType>
 void ParabolicProfileUtilities::ImposeParabolicProfile(
     ModelPart &rModelPart,
-    const TInputType& rMaxParabolaValue,
+    const TInputType & rMaxParabolaValue,
```

Comment:
```suggestion
    const TInputType& rMaxParabolaValue,
```
I think this is a debugging left over.

Reply:
Nop, this is needed, is the function or the value in the input :S

Reply:
But isn't it the same?

Reply:
aaa you mean the space, yes.